conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -3,11 +3,14 @@
 # !!!WARNING!!! This file MUST NOT be referenced in the "application.conf" file to avoid risk of rolling test routes in the production environment.
 # If you need test routes when running tests in CI make sure that the profile for this micro-service (used by service-manager) defines this router as parameter.
 # To do so add the following line to the micro-service profile: "-Dapplication.router=testOnlyDoNotUseInAppConf.Routes"
-# To start the micro-service locally using the test routes run the following command: "sbt run -Dapplication.router=testOnlyDoNotUseInAppConf.Routes" 
+# To start the micro-service locally using the test routes run the following command: "sbt run -Dapplication.router=testOnlyDoNotUseInAppConf.Routes"
 
 # Any test-only end-point should be defined here.
 # !!!WARNING!!! Every route defined in this file MUST be prefixed with "/test-only/". This is because NGINX is blocking every uri containing the string "test-only" in production.
 # Failing to follow this rule may result in test routes deployed in production.
 
++ nocsrf
+DELETE     /transit-movements-guarantee-balance/test-only/drop-mongo-collection       controllers.testOnly.TestOnlyMongoController.dropMongoCollection()
```

Comment:
Very small thing, but would it be possible to change this route to `/transit-movements-guarantee-balance/test-only/balances`?
I think the intent to drop the collection is already clear from the `DELETE` method, and this endpoint operates on balance requests.